src/js/AuthUI.js

```diff
@@ -39,19 +60,28 @@ class AuthUI {
 
     response
       .then((res) => {
-        console.log(res);
         if (res.status === 200) {
           localStorage.setItem('AUTH_TOKEN', res.data.jwt);
+          localStorage.setItem('USER_LOGIN', res.data.user.username);
```

Comment:
Work with local storage should be via class User

Reply:
Fixed